### PR TITLE
update gm log

### PR DESCRIPTION
**Pull request by HaoXuan40404**

### CODE REVIEW

File: libdevcrypto/gm/sm2/sm2.cpp [hunk 1]
```diff
@@ -34,34 +36,39 @@ bool SM2::genKey()
     sm2Group = EC_GROUP_new_by_curve_name(NID_sm2);
     if (!sm2Group)
     {
-        LOG(ERROR) << "Error Of Gain SM2 Group Object";
+        CRYPTO_LOG(ERROR) << "[#CRYPTO::SM2::genKey] Error Of Gain SM2 Group Object";
```
Comment (chaychen2005) [inline libdevcrypto/gm/sm2/sm2.cpp]: CRYPTO_LOG(ERROR) << "[genKey] Error Of Gain SM2 Group Object";  maybe better without ”#”.

File: libdevcrypto/gm/sm2/sm2.h [hunk 2]
```diff
@@ -21,27 +21,28 @@
...
  */
 #pragma once
-#include <iostream>
-#include <string>
-using namespace std;
 #include <openssl/sm2.h>
 #include <openssl/sm3.h>
+#include <iostream>
+#include <string>
+#define CRYPTO_LOG(LEVEL) LOG(LEVEL) << "[#CRYPTO] "
```
Comment (wheatli) [inline libdevcrypto/gm/sm2/sm2.h]: put the ERROR_OUTPUT into the definition of CRYPTO_LOG if you need to output error msg to std::out.

But I don't think you need

Comment (wheatli) [reply libdevcrypto/gm/sm2/sm2.h]: It's too ugly for logging twice in code

Comment (HaoXuan40404) [reply libdevcrypto/gm/sm2/sm2.h]: I just modified it according to the size of everyone, maybe we need to modify our standards?

Comment (HaoXuan40404) [reply libdevcrypto/gm/sm2/sm2.h]: like line 56~59 in SecureInitializer.cpp

File: libdevcrypto/gm/sm2/sm2.cpp (hunk 1 above)
Comment (HaoXuan40404) [reply libdevcrypto/gm/sm2/sm2.cpp]: get it

File: libdevcrypto/gm/sm2/sm2.h (hunk 2 above)
Comment (HaoXuan40404) [reply libdevcrypto/gm/sm2/sm2.h]: modify it now,maybe this can be better ; )

**Pull request merged**